formidable/views.py

```diff
@@ -290,3 +293,32 @@ def get_form(self, form_class):
 
     def get_form_kwargs(self):
         return {'data': self.request.GET}
+
+
+class ValidateViewFromSchema(ValidateView):
+    """
+    Acts like `ValidateView` but it uses a Formidable JSON schema instead of a
+    `Formidable` object.
+
+    """
+
+    settings_permission_key = 'FORMIDABLE_PERMISSION_USING'
+
+    def get_formidable_object(self):
```

Comment:
maybe one could add a docstring here to explain what's supposed to be implemented here.